src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -105,10 +105,12 @@ public String toString() {
 
         public boolean multitenancy_enabled = true;
         public String server_username = "kibanaserver";
+        public String opendistro_role = "kibana_server";
```

Comment:
Can we change to not have any default role (set to blank/null) ?

If we plan to add a default role, we should update default roles and roles_mapping configuration.

If users will want to use this feature, they'll have to manually update the domain configuration and create the required role and role_mapping

Similar for V6 config.